docs/products/alloydbomni/get-started.md

```diff
@@ -6,25 +6,53 @@ keywords: [quick start]
 
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
-import ConsoleLabel from "@site/src/components/ConsoleIcons"
-import CreateService from "@site/static/includes/create-service-console.md"
-import Help from "@site/static/includes/cli-help.md"
+import ConsoleLabel from "@site/src/components/ConsoleIcons";
+import CreateService from "@site/static/includes/create-service-console.md";
+import Help from "@site/static/includes/cli-help.md";
+import TerraformPrereqs from "@site/static/includes/terraform-get-started-prerequisites.md";
+import TerraformApply from "@site/static/includes/terraform-apply-changes.md";
+import TerraformSample from '@site/src/components/CodeSamples/TerraformSample';
 
 Start using Aiven for AlloyDB Omni by setting up a service and connecting to your new default database with a programming language of your choice.
 
 ## Prerequisites
 
-Depending on a dev tool to use for working with Aiven for AlloyDB Omni:
+<Tabs groupId="group1">
+<TabItem value="console" label="Console" default>
 
 - Access to the [Aiven Console](https://console.aiven.io)
-- [Aiven CLI](/docs/tools/cli)
-- [Aiven Provider for Terraform](/docs/tools/terraform)
-- [Aiven Operator for Kubernetes®](/docs/tools/kubernetes)
+- [psql](https://www.postgresql.org/download/) command line tool installed
+
+</TabItem>
+<TabItem value="cli" label="CLI">
+
+- [Aiven CLI](https://github.com/aiven/aiven-client) installed
+- [psql](https://www.postgresql.org/download/) command line tool installed
+
+</TabItem>
+<TabItem value="terraform" label="Terraform">
+
+- [Terraform installed](https://www.terraform.io/downloads)
+- A [personal token](https://docs.aiven.io/docs/platform/howto/create_authentication_token.html)
+- [psql](https://www.postgresql.org/download/) command line tool installed
+
+</TabItem>
+<TabItem value="k8s" label="Kubernetes">
```

Comment:
To make it work synchronically with the rest of the K8s tabs. Or you can fix the other three K8s tbs from `k8` to `k8s`, up to you :-)

```suggestion
<TabItem value="k8" label="Kubernetes">
```

Reply:
Those tabs get me every time 🤣  Thanks for catching that!